### PR TITLE
framework/st_things: Fix get filepath logic

### PR DESCRIPTION
**Pull request by KimDongEon**
Flexible logic to get file path for user authentication from json in Data Manager

The path to the file for user authentication may change during development, so change it flexibly to work even if it changes.

### CODE REVIEW

**Comment by seinfra** [comment]
Code Rule Check Result:
framework/src/st_things/things_stack/src/common/framework/things_data_manager.c:1197: ERROR: [IDT_M_TAB] code indent should use tabs where possible
framework/src/st_things/things_stack/src/common/framework/things_data_manager.c:1197: ERROR: [IDT_M_TAB] please, no space before tabs
framework/src/st_things/things_stack/src/common/framework/things_data_manager.c:1274: ERROR: [SPC_M_KWD] space required before the open parenthesis '('
framework/src/st_things/things_stack/src/common/framework/things_data_manager.c:1287: ERROR: [SPC_M_KWD] space required before the open parenthesis '('
framework/src/st_things/things_stack/src/common/framework/things_data_manager.c:1300: ERROR: [SPC_M_KWD] space required before the open parenthesis '('
framework/src/st_things/things_stack/src/common/framework/things_data_manager.c:1313: ERROR: [SPC_M_KWD] space required before the open parenthesis '('

File: framework/src/st_things/things_stack/src/common/framework/things_data_manager.c [hunk 1]
```diff
@@ -1260,23 +1263,71 @@ static int parse_things_info_json(const char *filename)
 				cJSON *certificate = cJSON_GetObjectItem(file_path, KEY_CONFIGURATION_FILEPATH_CERTIFICATE);
 				cJSON *privateKey = cJSON_GetObjectItem(file_path, KEY_CONFIGURATION_FILEPATH_PRIVATEKEY);
 
-				if (NULL != svrdb && NULL != provisioning && NULL != certificate && NULL != privateKey) {
-					memset(g_svrdb_file_path, 0, (size_t) MAX_FILE_PATH_LENGTH);
-					memset(g_certificate_file_path, 0, (size_t) MAX_FILE_PATH_LENGTH);
-					memset(g_private_key_file_path, 0, (size_t) MAX_FILE_PATH_LENGTH);
+				if (NULL == svrdb || NULL == provisioning || NULL == certificate || NULL == privateKey) {
+					return 0;
```
Comment (JeongJunSik) [inline framework/src/st_things/things_stack/src/common/framework/things_data_manager.c]: add Error Log

Comment (KimDongEon) [reply framework/src/st_things/things_stack/src/common/framework/things_data_manager.c]: modity

**Comment by seinfra** [comment]
Code Rule Check Result:
framework/src/st_things/things_stack/src/common/framework/things_data_manager.c:1197: ERROR: [IDT_M_TAB] code indent should use tabs where possible
framework/src/st_things/things_stack/src/common/framework/things_data_manager.c:1197: ERROR: [IDT_M_TAB] please, no space before tabs
framework/src/st_things/things_stack/src/common/framework/things_data_manager.c:1275: ERROR: [SPC_M_KWD] space required before the open parenthesis '('
framework/src/st_things/things_stack/src/common/framework/things_data_manager.c:1288: ERROR: [SPC_M_KWD] space required before the open parenthesis '('
framework/src/st_things/things_stack/src/common/framework/things_data_manager.c:1301: ERROR: [SPC_M_KWD] space required before the open parenthesis '('
framework/src/st_things/things_stack/src/common/framework/things_data_manager.c:1314: ERROR: [SPC_M_KWD] space required before the open parenthesis '('

**Pull request closed without merging**